base/db/tests/Rcheck_reference.log

```diff
@@ -34,98 +90,49 @@ Requires orphaned package: ‘udunits2’
 * checking whether the namespace can be loaded with stated dependencies ... OK
 * checking whether the namespace can be unloaded cleanly ... OK
 * checking loading without being on the library search path ... OK
-* checking dependencies in R code ... OK
+* checking use of S3 registration ... OK
+* checking dependencies in R code ... NOTE
+Namespaces in Imports field not imported from:
+  ‘glue’ ‘knitr’
```

Comment:
If I interpret this correctly, does this mean this version would allow this new NOTE? @infotroph is that what we want?